### PR TITLE
fix(a11y): custom prop for aria-describedby to match native prop

### PR DESCRIPTION
**Pull request by mhewson**
Not huge, but confusing to have custom props deviate from the native name

### CODE REVIEW

File: src/components/modal/CdrModal.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default {
       required: false,
       default: true,
     },
-    ariaDescribedbBy: {
```
Comment (mhewson) [inline src/components/modal/CdrModal.jsx:32]: also a sadness typo

Comment (cowills) [reply src/components/modal/CdrModal.jsx:32]: im pretty sure we can remove this prop entirely, unless we need special access to it there's no need to make props for native attrs. plus if the prop has been typo'd this whole time then... 😆

Comment (mhewson) [reply src/components/modal/CdrModal.jsx:32]: 👍 yea probably should just go

File: src/components/modal/CdrModal.jsx [hunk 2]
```diff
@@ -64,7 +59,6 @@ export default {
   computed: {
     dialogAttrs() {
       return {
-        'aria-describedby': this.ariaDescribedBy,
```
Comment (cowills) [inline src/components/modal/CdrModal.jsx:67]: you still need to add this here, but you can pull the prop off attrs instead

$atttrs['aria-describedby'] should do it

**Comment by cowills** [comment]
I will pull this into the safari modal bugfix work

**Pull request closed without merging**